server/data_anndata/anndata_adaptor.py

```diff
@@ -310,16 +309,15 @@ def compute_embedding(self, method, obsFilter):
             raise FilterError("Error parsing filter")
         with ServerTiming.time("layout.compute"):
             X_umap = scanpy_umap(self.data, obs_mask)
```

Comment:
You are no longer normalizing the embedding here.  Is it no longer needed?

Reply:
correct, not needed.  That happens when the client requests the embedding via the normal GET /layout/obs route (just like all other embeddings).

Reply:
sounds good.